setup.cfg

```diff
@@ -28,6 +28,6 @@ exclude = mistune,doc/conf.py
 [bumpversion:file:.zenodo.json]
 
 [tool:pytest]
-filterwarnings =
+filterwarnings = 
```

Comment:
Is this space needed?

Reply:
It's added automatically by `bumpversion`. I added this file to the pre-commit "ignore whitespace" list so pre-commit passes regardless. This has flipped between "space" and "no space" a lot of times.

Reply:
Ah well it is what it is.